packages/vm/typedoc.js

```diff
@@ -1,10 +1,9 @@
 module.exports = {
   extends: '../../config/typedoc.js',
-  entryPoints: ['lib'],
   exclude: [
     'tests/**/*.ts',
-    'lib/bloom/*.ts',
-    'lib/evm/**',
-    'lib/state/cache.ts',
+    'src/bloom/*.ts',
+    'src/evm/**',
+    'src/state/cache.ts',
```

Comment:
Ah, important catch, I really thought I had it. Thanks for having another closer look here.